<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMyPortfolio</class>
 <widget class="QDialog" name="AdminMyPortfolio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1344</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="yearLineEditAdminPortfolio">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="monthLineEditAdminPortfolio">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>År</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Månad</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="marketValueLineEditAdminPortfolio">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Marknadsvärde</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Insättning</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addedMoneyLineEditAdminPortfolio">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Utdelning</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ReceivedDividentLineEditAdminPortfolio">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addDataPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Lägg till</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>1121</width>
     <height>591</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="removeDataPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Radera</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
